src/components/AllComments.jsx

```diff
@@ -1,10 +1,28 @@
 import moment from "moment";
 import { useState, useEffect } from "react";
 import { getComments } from "../utils/api";
+import { deleteComment } from "../utils/api";
+import React from "react";
 
 export const AllComments = ({ review }) => {
   const [allComments, setAllComments] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
+  const refreshPage = () => {
+    window.parent.location = window.parent.location.href;
+  };
+  let username = "jessjelly"
+
+  const handleClick = (comment) => {
+    return function () {
+      if (window.confirm("Comment will be deleted, is that okay?")) {
+        deleteComment(comment.comment_id).then((res) => {
+          setAllComments((currState) => [...currState]);
+          alert("Comment deleted");
```

Comment:
This `setState` call isn't doing anything - you're just updating the state with an exact shallow copy of the array!